ci-scripts/build-deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+echo "Building SPA core lib"
+yarn build:core:lib
+echo "-----"
+echo "Building SPA app"
+ng build storefrontapp --prod -c=ci --base-href "https://sap.github.io/cloud-commerce-spartacus-storefront/"
```

Comment:
Do we want to have the production build (minified and uglyfied) here?
If there's an error in the console, the stack trace would be messed up, and it would be difficult to troubleshot potential problems discovered here

Reply:
Done.